src/ansys/geometry/core/_grpc/_services/v0/repair_tools.py

```diff
@@ -208,9 +216,16 @@ def find_small_faces(self, **kwargs) -> dict:  # noqa: D102
     def find_stitch_faces(self, **kwargs) -> dict:  # noqa: D102
         from ansys.api.geometry.v0.repairtools_pb2 import FindStitchFacesRequest
 
-        request = FindStitchFacesRequest(faces=kwargs["faces"])
+        request = FindStitchFacesRequest(
+            faces=kwargs["faces"],
+            maximum_distance=DoubleValue(value=kwargs["distance"].value)
```

Comment:
If you are using Distance/Angle objects  make sure to use the proper conversion methods. They should be in the ``base`` subpackage inside ``_grpc``. They handle consistently the units conversion to the server units

Reply:
Applies everywhere in this PR